package.json

```diff
@@ -11,8 +11,9 @@
     "dist/"
   ],
   "scripts": {
-    "build": "rm dist/* && tsc",
-    "test": "yarn build && mocha"
+    "build": "rm -rf dist/* && tsc",
```

Comment:
With the `-rf` can we change `dist/*` to just `dist/`?

Reply:
Ah, yeah we can because `tsc` will create the folder if it doesn't exist, although this way we won't unnecessarily delete it.